### DISCUSSION TITLE
How can I find out the name of a folder icon?

### DISCUSSION THREAD

**Discussion by BaughBerry** [FlorianWoelki/obsidian-icon-folder#339] [Q&A] [answered]
I chose a folder icon, but the icons in the pack I'm using (Tabler Icons) are not always intuitively named, so I can't remember what it was called. Is there a way to find out the name of the icon somehow?

I've also attached a quick picture of the icon in case anyone knows what it's called. Thanks!

<img width="219" alt="Screenshot 2024-01-03 at 3 56 20 PM" src="https://github.com/FlorianWoelki/obsidian-iconize/assets/129925078/8dacb957-f24a-46db-a02b-d57848fc4b13">

**Comment by FlorianWoelki** [comment, marked as answer]
You can always retrieve the name of the icon in the saved `data.json` file of the plugin.

If you are not comfortable with this, there is currently no way to view the name of the icon. But it would be a good enhancement!
